recipes/mender-app-update-module/debian-master/control

```diff
@@ -0,0 +1,15 @@
+Source: mender-app-update-module
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-app-update-module
+Architecture: all
+Depends: ${misc:Depends}
+Suggests: mender-client
```

Comment:
I think it should `depend` on the client. Can the update module run as standalone in any scenario?

Also, please add the minimum version. In theory it could work with >2.0 (when we introduced update modules) but I would write the oldest version that we have tested it against.

Reply:
fixed, and also: this meant updating the test. I did Suggests: on purpose, but if you think it should be Depends: I am ok with it.